public/config/configuration.json

```diff
@@ -15,6 +15,10 @@
     "AUTH_TOKEN_URL": "https://dev.oidc.gov.bc.ca/auth/realms/fcf0kpqr/protocol/openid-connect/token",
     "QUICK_SEARCH_PUBLIC_ID": "",
     "QUICK_SEARCH_PUBLIC_SECRET": "",
-    "ENTITY_SELECTOR_URL": "https://entity-selection-dev.pathfinder.gov.bc.ca/"
+    "HOTJAR_ID": "2284327",
```

Comment:
This is the Hotjar ID for local dev. There area other ones for Dev, Test and Prod.